seaborn/categorical.py

```diff
@@ -3889,14 +3891,15 @@ def catplot(
         Categorical variables that will determine the faceting of the grid.
     {col_wrap}
     {stat_api_params}
-    {order_vars}
+    {order_vars} ``order`` is ignored if the ``share`` parameter for the
+        categorical axis is ``False``.
     row_order, col_order : lists of strings, optional
         Order to organize the rows and/or columns of the grid in, otherwise the
         orders are inferred from the data objects.
-    kind : string, optional
+    kind : {{"strip", "swarm", "box", "violin", "boxen", "point", "bar", \
```

Comment:
Given that the enumeration of possible values isn't used, I think I'd rather have them in the description if there are too many to fit on one line.

Two related comments:
- I'm gradually removing "optional" from the docstrings, as basically every parameter of the plotting functions is optional and it just adds noise.
- The pyflakes config now allows lines up to 88 characters, per the Black recommendations, so there might be space

Reply:
Thanks! Moved the possible values back to the description.
Thanks for the comments, will use them.